# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../musl/pkg/util-vserver/fd_set.patch
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- util-vserver-0.30.216-pre3054/src/vshelper-sync.c.orig	2013-10-31 14:44:26.000000000 +0200
+++ util-vserver-0.30.216-pre3054/src/vshelper-sync.c	2014-03-14 22:34:56.983716281 +0200
@@ -28,6 +28,7 @@
 #include <signal.h>
 #include <stdio.h>
 #include <assert.h>
+#include <sys/select.h>
 
 static void
 showHelp(int fd, char const *cmd, int res)

